packages/blockchain-wallet-v4-frontend/src/modals/BuySell/PaymentMethods/Methods/PaymentCard/index.tsx

```diff
@@ -57,7 +57,7 @@ const PaymentCard: React.FC<Props> = ({ icon, onClick, text, value }) => (
     <Content>
       <DisplayTitle>{text}</DisplayTitle>
       <DisplaySubTitle>
-        <FormattedMessage id='copy.instantly_available' defaultMessage='Instantly Available' />
+        <FormattedMessage id='copy.instantly_available' defaultMessage='Buy Small Amounts' />
```

Comment:
could you also change the id name? so it refers to what the default message means